### PR TITLE
Use MYSQL_PORT in setup

### LINKED ISSUE #341: MYSQL_PORT is ignored in setup
### Describe the bug

MYSQL_PORT setting is ignored in setup.

### To Reproduce

Set MYSQL_PORT to some value other than the default 3306 when starting the docker image. Setup tries to connect to MySQL on port 3306 instead of provided MYSQL_PORT.

### Expected behavior

Port provided should be used to connect to MySQL.

### Client configuration

- OpenEMR version: 7.0.1
- Operating system: Alpine (in Docker)

### PR DESCRIPTION
**Pull request by eugeneiiim**
Fixes #341 

#### Short description of what this resolves:
MYSQL_PORT setting being ignored

#### Changes proposed in this pull request:
Add `port=` configuration in `prepareVariables` when `MYSQL_PORT` is set.

### CODE REVIEW

**Comment by jesdynf** [comment]
Oh, goodness, thank you. I can see how we would've missed this but it's nice to have it covered.

This is adjusting the devtools configuration script but not the container setup itself, or any of the docker-compose scripts that launch the container. Is it necessary to review those as well or is the problem you're trying to solve strictly within the devtools?

File: docker/openemr/7.0.1/utilities/devtoolsLibrary.source [hunk 1]
```diff
@@ -23,6 +23,9 @@ prepareVariables() {
     else
         CUSTOM_DATABASE="openemr"
     fi
+    if [ "$MYSQL_PORT" != "" ]; then
+        CONFIGURATION="${CONFIGURATION} port=${MYSQL_PORT}"
+    fi
```
Comment (bradymiller) [inline docker/openemr/7.0.1/utilities/devtoolsLibrary.source:28]: would rec following how is done in the $MYSQL_DATABASE block above where is converted to $CUSTOM_DATABASE and the default port of 3306 is set if not given. and would then use the $CUSTOM_MYSQL_PORT setting below.

**Comment by bradymiller** [comment]
@eugeneiiim , Thanks for noticing this!
@jesdynf , that first change in the preparevariables function is used on the standard docker openemr configuration, so this will now allowing setting a port in the standard docker install (ie. will be a new setting at `Optional settings for auto installation are:` section in https://hub.docker.com/r/openemr/openemr/ instructions).

File: docker/openemr/7.0.1/utilities/devtoolsLibrary.source (hunk 1 above)
Comment (eugeneiiim) [reply docker/openemr/7.0.1/utilities/devtoolsLibrary.source:31]: Updated

**Comment by bradymiller** [comment]
hi @eugeneiiim , looks good and bringing in. thanks for the improvements! (fyi, also gonna bring your changes into the other supported dockers, 7.0.0, flex-edge, flex-3.17, flex-3.15-8)

2DV-2(13)

**Pull request merged**